CHANGES.md

```diff
@@ -1,3 +1,31 @@
+## edge-19.1.3
+
+* Controller
+  * Added `linkerd install-sp` command to install service profiles into the
+    linkerd control plane
+  * Improved API for ListPods (thanks @alenkacz!)
+  * Fixed `GetProfiles` API call not returning immediately when no profile
+    exists (resulting in proxies logging warnings)
+* Web UI
+  * Improved resource detail pages now show all resource types
+  * Fixed stats not appearing for routes that have service profiles installed
+* CLI
+  * Removed `--proxy-bind-timeout` flag from `linkerd install` and `linkerd inject`
+    commands, as the proxy no longer accepts this environment variable.
+  * Improved CLI appearance on Windows systems
+  * Improved `linkerd check` output, fixed some bugs in check
+  * Improved `linkerd stat` now supports DaemonSets (thanks @zknill!)
+  * Fixed panic when routes is called in single-namespace mode
+* Proxy
```

Comment:
Can you remove the trailing period from all of the bullets in this section? Sorry, @hawkw :-/